apps/app_mixmonitor.c

```diff
@@ -817,18 +840,32 @@ static void *mixmonitor_thread(void *obj)
 			|| mixmonitor_autochan_is_bridged(mixmonitor->autochan)) {
 			ast_mutex_lock(&mixmonitor->mixmonitor_ds->lock);
 
+			/* Skip writing audio for the first N seconds */
+			if (mixmonitor->skip_seconds > 0.0) {
+				struct timeval now = ast_tvnow();
+				double elapsed = ast_tvdiff_ms(now, skip_start) / 1000.0;
+
+				if (elapsed < mixmonitor->skip_seconds) {
+					ast_mutex_unlock(&mixmonitor->mixmonitor_ds->lock);
+					/* Skip this frame and continue */
+					goto frame_cleanup;
+				} else if (!skip_done_logged) {
```

Comment:
skip_done_logged doesn't need to exist. Once skipped you can set mixmonitor->skip_seconds to 0.